packages/variants/src/variants.ts

```diff
@@ -88,24 +88,36 @@ export function getNeutralVariant(theme: IPartialTheme): ITheme {
     bodyText: p.neutralPrimary,
     bodySubtext: p.neutralSecondary,
     bodyDivider: p.neutralQuaternaryAlt,
+    focusBorder: p.neutralSecondary,
```

Comment:
shouldn't this be focusBorder inner/outer?

Reply:
at least currently focus borders are in 2 parts

Reply:
@betrue-final-final - can you please address this question about focus border?

The slots affected by this PR matches the list of semantic slots that was put together by the designers, that has been triple checked and verified multiple times. (Also, I've heard that the double border is slated to go away soon, but I can't speak directly to that.)

Reply:
I think this got resolved, but if not I don't think I have an opinion on the method, just the result.